### ISSUE TITLE
It Takes Two (1426210)

### ISSUE THREAD

**Issue by knikkinen** [ValveSoftware/Proton#4696]
# Compatibility Report
- Name of the game with compatibility issues: It Takes Two
- Steam AppID of the game: 1426210

## System Information
- GPU: AMD RX 6900XT
- Driver/LLVM version: Mesa 21.1.0-devel (git-1ab309377e)
- Kernel version: 5.11.7-139-tkg-upds
- Link to full system information report: https://gist.github.com/Shadistikk/f4f802c1811664b2530ff80ea062fb7e
- Proton version: Proton Experimental / Proton 5.13-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms
Game launches fine, but cannot invite friends as game is dependent of Origin Overlay.

## Reproduction
1. Start game

2. Join Online

3. As you try to invite friends it complains Origin overlay is not enabled, thus you cannot invite anyone.

## Other

If needed i can provide logs, but as this has been a known problem that Origin Overlay hasn't work for
quite sometime i don't see any need for them as of right now.

**Comment by rejexy** [comment]
Do you have a Steam version or Origin? You can try to work around this with the origin lan emulator (But then you need to use a program like hamachi or zero-tier). It looks like Steam Remote Play Together is the only solution at the moment.

**Comment by knikkinen** [comment]
> Do you have a Steam version or Origin? You can try to work around this with the origin lan emulator (But then you need to use a program like hamachi or zero-tier). It looks like Steam Remote Play Together is the only solution at the moment.

Steam version. Yeah, it looks like Remote Play Together is now only solution, but as im using wayland currently clients cannot see only but black screen, and i know this is because of wayland.

**Comment by knikkinen** [comment]
https://github.com/doitsujin/dxvk/issues/1996 Looks like good news

**Comment by knikkinen** [comment]
Tested by applying current DXVK master to Proton-GE 6.1 and overlay works. With Proton Experimental even steam overlay doesn't load when dxvk master is applied.

Thanks Paul, Josh and Philip.

**Comment by nyanloutre** [comment]
Hello,

I am using Proton 6.3-2 and the Origin launcher is only launching for a few seconds before crashing and the game doesn't start. I enabled logs if it can be useful.

[steam-1426210.log](https://github.com/ValveSoftware/Proton/files/6275420/steam-1426210.log)

**Comment by parkerlreed** [comment]
@Shadistikk Using Lutris with 6.1 GE and DXVK master applied to the prefix I'm still getting no overlay in game (DXVK disabled in Lutris game settings). Any idea? (Foolishly didn't buy it through Steam)

**Comment by knikkinen** [comment]
> @Shadistikk Using Lutris with 6.1 GE and DXVK master applied to the prefix I'm still getting no overlay in game (DXVK disabled in Lutris game settings). Any idea? (Foolishly didn't buy it through Steam)

Might as well try it with wine-tkg if thats solves your case, otherwise i can't think what would be causing this other than DXVK is disabled in lutris settings.

E: Also just to confirm, did you compile DXVK from source?

**Comment by parkerlreed** [comment]
I used the nightly build artifacts. I'll take another stab later, thanks.

**Comment by aeikum** [comment]
The Origin overlay should be functional in today's Proton Experimental release, so hopefully It Takes Two is now playable with friends.

**Comment by Eelviny** [comment]
I'm able to run the game pretty smoothly so far, nice. With 6.5-GE-2 I'm able to get the overlay in-game.

The major issue I'm having right now is Steam Input not functioning in-game - I own Dualshock 4 controllers. From what I've read you can get XBox controllers working if you disable steam input, but naturally I need this to make my dualshock controllers work.

Edit: Steam input works with the Steam overlay enabled as of 6.3-3, I believe that the game has native dualshock support but with this method they show up as xbox controllers

**Comment by eth0OS** [comment]
Hello everyone, I've played 4 hours of It Takes Two without any problem on Proton 6.3-2, allowing my partner to joining my game from his origin client. However, since the Proton update to 6.3-3 I keep getting an error from Origin everytime I try to launch the game, it was working fine before.
Here are the [proton log](http://sprunge.us/wkQruz) of the game and some info about [my setup](http://sprunge.us/uLAzUZ).
Thank you!

**Comment by Eelviny** [comment]
@eth0OS I've had some issues when the Origin client tries to update itself, try switching to Proton 4.11, letting the game start once, then switching back to latest version

**Comment by eth0OS** [comment]
> @eth0OS I've had some issues when the Origin client tries to update itself, try switching to Proton 4.11, letting the game start once, then switching back to latest version

Thank you for your suggestion, I've deleted the prefix and tried to install it with Proton 4.11-13, but it will hang on Installing OriginSetup for hours.

I am open for other ideas!

I see in the logs that I have this exactly same issue: https://github.com/ValveSoftware/Proton/issues/3937
Origin runs fine under my system's wine 6.8.

## edit1

Kinda solved it. I've deleted the original pfx, and run the game again with proton 5.0-10 and these launch options: 

> PROTON_NO_ESYNC=1 PROTON_USE_WINED3D=1 %command%

I don't really know what they do but they are my first launch options to go when I face problems with proton.

I've let the game run once and them bumped the proton's version to 6.3-3 and kept the launch options there.

Ok, running the game with PROTON_USE_WINED3D=1 turn the most of the textures into a black mess! I'm now able to play with Proton 5.0-10 with no additional parameters, guess this will be the way till I figure how to fix it!

## edit2

Forcing RADV on Proton 6.3-3 I get an UE4 fatal error: [here is the proton log](http://sprunge.us/7UJ0In)
Using standard AMDVLK on Proton 6.3-3 I get the origin error I was talking about in the beginning: [proton+amdvlk log](http://sprunge.us/emgSHB)

## edit3

Proton 6.3-4 fixed the issue!

**Comment by iMonZ** [comment]
I tried this and the game works perfectly fine except that you can only connect one controller (and a keyboard) If I try to connect a second controller it's not detected. I tried with an Xbox One controller and Sunshine (Moonlight) Input.

EDIT:

I tested it on Lutris-wine-6.0-x86_64 and the game worked there but with wine-6.8 the game crashes with a fatal error.

EDIT 2:
I could fix the controller problem with the x360ce XInput 9.1.0 mode and I can use two Xbox One Controller at the same time, but now I have a new problem:
If a controller disconnects the game cant recognize the controller until I restart the whole game.
If someone has a workaround for that It would be welcome.

**Comment by eth0OS** [comment]
> EDIT:
> 
> I tested it on Lutris-wine-6.0-x86_64 and the game worked there but with wine-6.8 the game crashes with a fatal error.

Hello, would mind elaborating on how you launched the game with lutris-wine? I've tried Proton and Proton-GE already to no avail, I'd like to try running the game with another build!

**Comment by Eelviny** [comment]
I've got this game running pretty well now. I have 2x Dualshock 4 controllers using Steam Input to fake Xbox controllers. Game runs smooth but I get the occasional sound issue where it cuts out for half a second. On the newest Proton versions, the Origin overlay works great and I can play online too.

Here's my setup for those interested in trying to replicate it:

## System specs:

- Dell XPS 13 9300 laptop with Thunderbolt 3 eGPU
- Fedora 34, `kmod-nvidia` version 465.27
- Intel i7 1065G7 8-core
- NVIDIA GTX1070 FE

## Game settings:

- Proton 6.8-GE-1 (remember to delete wine prefix in compatdata before upgrading to this version)
- Steam overlay on (needed for steam input to work)
- Steam input on (leave it at defaults if it's already globally on, no need to force on)
- Command: `PROTON_NO_ESYNC=1 %command%` (seems to help with audio issues but not 100% sure of this, it might also tank performance in busy sections of the game)

One thing to note is that the game uses the Origin launcher. While the game will remain fairly static and unchanging, the launcher can update all the time, breaking itself in the process. I got the game installed using Proton 4.11, and haven't tested later versions. On 6.8-GE-1 I can launch Origin and play the game just fine.

**Comment by davispuh** [comment]
Using latest Proton Experimental  `1623850434 experimental-6.3-20210616`
Origin crashes for some reason, but I found out that if you start it manually it works.
Next issue I see is that seems there's missing GStreamer plugins

```
winegstreamer: error: decodebin0: Your GStreamer installation is missing a plug-in.
winegstreamer: error: decodebin0: ../src-gst_base/gst/playback/gstdecodebin2.c(4719): gst_decode_bin_expose (): /GstBin:bin0/GstDecodeBin:decodebin0:
no suitable plugins found:
Missing decoder: MPEG-4 AAC (audio/mpeg, mpegversion=(int)4, framed=(boolean)true, stream-format=(string)raw, level=(string)2, base-profile=(string)lc, profile=(string)lc, codec_data=(buffer)119056e500, rate=(int)48000, channels=(int)2)
Missing decoder: H.264 (High Profile) (video/x-h264, stream-format=(string)avc, alignment=(string)au, level=(string)4, profile=(string)high, codec_data=(buffer)01640028ffe1001e67640028acd940780227e5c05a808080a000000300200057e401e30632c001000668e9386cb22c, width=(int)1920, height=(int)1080, framerate=(fraction)60/1, pixel-aspect-ratio=(fraction)1/1)
```
And then it gets stuck on black screen.

With Proton 6.3 I get same missing plugin message but I got it running few times but only sometimes, not sure how so.

**Comment by davispuh** [comment]
It think I figured it out, it seems it was caused by `Origin In-Game` settings, after disabling it works fine. Strangly if it's disabled then Origin features still are working, like can invite friends and such

**Comment by peterge1998** [comment]
> Origin crashes for some reason, but I found out that if you start it manually it works.

 How do i do this? For me it does not launch using neither latest proton-ge 6.10 or latest proton experimental...
Using flatpak steam

**Comment by kisak-valve** [comment]
Hello @peterge1998, starting with Proton 5.13, Proton is run inside the Steam Linux Runtime - Soldier container environment. The runtime environment currently needs a bleeding edge version of Flatpak if you want to Proton with Flatpak'd Steam. Please give https://github.com/ValveSoftware/steam-runtime/blob/master/doc/steamlinuxruntime-known-issues.md#flatpak a read.

**Comment by davispuh** [comment]
> > Origin crashes for some reason, but I found out that if you start it manually it works.
> 
> How do i do this? For me it does not launch using neither latest proton-ge 6.10 or latest proton experimental...
> Using flatpak steam

I didn't use container. I just did
```
export STEAM_COMPAT_CLIENT_INSTALL_PATH=~/.steam/steam
export STEAM_COMPAT_DATA_PATH=~/.steam/steam/steamapps/compatdata/1504980/
~/.steam/steam/steamapps/common/Proton\ 6.3/proton run ~/.steam/steam/steamapps/compatdata/1504980/pfx/drive_c/Program\ Files\ \(x86\)/Origin/Origin.exe
```

**Comment by cirelli94** [comment]
I'm unable to use the steam controller so I can't play in local with 2 players, one with the controller, one with mouse and keyboard.

**Comment by Holzhaus** [comment]
I'm able to reach the main menu without issues, but since I'm trying to play this via Steam Link, both controllers are detected as keyboard and mouse, which makes local co-op impossible.

**Comment by cirelli94** [comment]
I've been able to play in local with 2 players with k/m and one controller disabling Steam Input for this game.

You should try to disable Steam Input as well @Holzhaus with two controllers.

**Comment by Holzhaus** [comment]
> You should try to disable Steam Input as well @Holzhaus with two controllers.

That works when I sit at my desk in front of my computer, but Steam Input is required for Remote Play/Steam Link.

Steam Input seems to work with Proton-6.21-GE-2 though. Unfortunately it broke screen grabbing, but that is easily fixable via gamescope.

**Comment by nyanloutre** [comment]
I still have the same problem of Origin starting and crashing before the game launch with any recent proton version (6.3-8, 6.21-GE-2 and experimental)


[steam-1426210.log](https://github.com/ValveSoftware/Proton/files/7613963/steam-1426210.log)

**Comment by CvRXX** [comment]
I get an unknown exception (code=6ba).

I'm running `PROTON_LOG=1 PROTON_NO_ESYNC=1 PROTON_USE_WINED3D11=1 %command%`. 

The game going through shaders and then the dialog disappears and nothing happens. The steam button returns to the 'play' state. 

Log: 
[steam-1426210.log](https://github.com/ValveSoftware/Proton/files/7788049/steam-1426210.log)

**Comment by Paiusco** [comment]
Trying to run it out of the box on Steam, using Proton 6.3-8 but it doesn't even starts. The dialog box loads, nothing happens and the steam button gets back to 'play' state

**Comment by cirelli94** [comment]
> Trying to run it out of the box on Steam, using Proton 6.3-8 but it doesn't even starts. The dialog box loads, nothing happens and the steam button gets back to 'play' state

Are you using Steam via Flatpak? I had this problem with Flatpak. 
If Steam is installed with dnf (on Fedora), it works.

**Comment by Paiusco** [comment]
> > Trying to run it out of the box on Steam, using Proton 6.3-8 but it doesn't even starts. The dialog box loads, nothing happens and the steam button gets back to 'play' state
> 
> Are you using Steam via Flatpak? I had this problem with Flatpak. If Steam is installed with dnf (on Fedora), it works.

No, I'm using Steam from directly from pacman :( At first I though it was related to had being installed in my HD (NTFS), trying from scratch on my main ssd (ext4) and no changes. Wondering what else I should try

**Comment by theli-ua** [comment]
This might've worked previously but today this gets stuck at origin thin install, if you kill it and eventually get it to install somehow it crashes 9/10 times, if you spend few hours and somehow end up with origin running the game still won't start unfortunately. I wonder if some Origin update just broke this. 
Which is all weird given that its marked as playable on steamdeck

**Comment by mpiorowski** [comment]
@Paiusco have the same problem, using manjaro. The dialog box loads, get "OrigintThiinSetup", nothing happens and the steam button gets back to 'play' state

**Comment by Paiusco** [comment]
which DE/WM are you using? 
I was struggling to make it work on Qtile, however when I decided to try Openbox, all went fine. Maybe it's worth reinstalling from scratch on a non-tiling WM if that's your case

**Comment by FeralBytes** [comment]
Linux Mint 20.3 tried to play today with Experimental and 7.0-5 and neither would run the game on my AMD GPU. On my wife's Nvidia GPU it managed to get her to the Origin create account screen but not beyond that. I believe the latest update to the game ruined what ever progress we had made.

**Comment by Bob0111** [comment]
With the new EA app, I can only start the game with proton experimental and proton-GE. I can use them, and the EA app overlay works just fine. But the steam overlay doesn't work. I hope in the future Valve could fix the problem and make the steam overlay work again because the EA app overlay provide nothing expect the chat and inviting friends function.

**Comment by mercuriete** [comment]
Same problem like @Bob0111 
The game open perfectly with Proton Experimental but the steam overlay doesn't work and the gamepad is using the controls like I were in desktop instead of normal gamepad controls.
It works with keyboard but that is not ideal.

TLDR; No steam overlay. Steam Input behaving bad.

**Comment by kisak-valve** [comment]
Related: #4009

**Comment by mercuriete** [comment]
I just updated the steam client and with the new big picture mode, the Steam Input with ps4 gamepad started working again making it playable.
But it still fails to open the new big picture overlay in game.
TLDR: No Steam overlay. New big picture fixes Steam Input.

**Comment by rufreakde** [comment]
Proton Experimental seems to launch thE EA customize Setup screen for me but not able to press „Lets go“ … no input works.

I am using latest chimera OS. 

Update: I had to connect a mouse and press the button with it… controller is not working in the EA App…

**Comment by poulet2vicecity** [comment]
I think an auto update broke the game for me...
I was running the game with Proton Experimental on ChimeraOS (Arch-based linux OS) and never had any problem in 4 hours of gaming over a week (several booting). One day the game just refused to launch.
I tried many things :

- Switching between Proton versions (also GE versions)
- Trying the launch option `gamemoderun %command%`
- Reinstalling the whole game
- Disabling steam overlay
- Removing the compatdata folder
- Disabling Steam Shader Pre-Caching

Some Proton version made the EA apps appearing on the screen but it keep failing to launch the game and just go back to steam... I'm out of ideas...

We are in the beggining of the game and it's really sad/annoying that the game broked itself with no reason. I'm pretty sure it's the useless EA app that cause the issue. I hate EA and avoid their games but this one really tempted me and my partner...
Does anyone could help us getting it back to a playble state ?

**Comment by bjornfor** [comment]
> I think an auto update broke the game for me...
> ...
> * Reinstalling the whole game
> ...
>Does anyone could help us getting it back to a playble state ?

I've had the same problem happen twice now. It seems to be the EA Origin program that breaks the game. I wiped ~/.local/share/Steam on my not-main-machine and re-installed the game. Works fine! I think somewhere EA Origin leaves some state that eventually breaks it :-( It would be nice to know exactly where that state is, so the game can be fixed without resetting *all* userdata.

**Comment by mercuriete** [comment]
> So I have to reset most of my gaming rig in order to make it work ?

I am not 100% sure but EA app have cloud savegame for this game.
So in theory you could delete proton/wine profile and the savegame should be downloaded automatically from ea cloud the next time you play.
(disclaimer: please do a backup of your savegame before doing experiments like this).

Yesterday I could play with a brand new Dual Sense gamepad but the problem is when a player invites you to play, you have to use the keyboard key "Y" to accept the invitation.
After you accept the invitation using the keyboard, the game works flawlesly.

**Comment by poulet2vicecity** [comment]
I have completely reinstall steam and the game... still don't work... I might just install it on Windows...

**Comment by bjornfor** [comment]
> I have completely reinstall steam and the game... still don't work... I might just install it on Windows...

Did you remove `~/.local/share/Steam`? If you didn't then EA Origin state was not reset and is probably why it failed again.

Btw, I planned doing another attempt at finding a more specific path in `~/.local/share/Steam` to reset *just* the EA Origin state, but after a Steam automatic update and installing the game again (which I un-/in-/uninstalled last week during a debug session), it works now! I guess updating/using a new Proton version cause Origin to be re-installed/start clean?

**Comment by poulet2vicecity** [comment]
I did and steam asked to be reinstall so did I but it didn't solve the problem at all.

Just changing the Proton version didn't help for me as I mentionned in my first message :'(
What Proton version are you using ?

**Comment by bjornfor** [comment]
> Just changing the Proton version didn't help for me as I mentionned in my first message :'( What Proton version are you using ?

It used Proton Experimental by default (even though my global Steam setting says 7.0-6). I also tested it (successfully) with Proton 7.0-6.

**Comment by howels** [comment]
Completely borked on latest proton, likely due to the EA launcher. Tried 7.0-6, 8.0-4 experimental and a couple of GE versions. EA client launches, says its launching the game then after a minute of nothing "the app crashed unexpectedly".

Ubuntu 22.04 LTS
Kernel 6.6.5
NVIDIA 545.29.06
Proton Experimental

[steam-1426210.log](https://github.com/ValveSoftware/Proton/files/14229294/steam-1426210.log)

**Comment by howels** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4696#issuecomment-1936994225

Found a fix - using ProtonTricks with Proton Experimental I installed `d3dcompiler_47` and the `liberation` font package which allowed the game to launch.
Credit to https://steamdeckhq.com/tips-and-guides/fixing-ea-play-blank-screen-for-ea-games-on-steam/

**Comment by ye-5** [comment]
I cant get the game up and running on the client with remote play. I want to stream the game from my main pc to a device in an other room. Both devices have Linux (Arch - Host, Manjaro - Client). But i also tried it with ubuntu 22.04 on both and still failed.
The game starts on the host and runs flawless, but the screen of the client stays black.

Any ideas how i can fix this?

**Comment by cirelli94** [comment]
@ye-5 that isn't a game problem, but rather a steam client problem, see https://github.com/ValveSoftware/steam-for-linux/issues/6148

**Comment by kisak-valve** [comment]
EA friends not showing up

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7677.
@legoraft posted on 2024-04-28T11:40:59:

# Compatibility Report
- Name of the game with compatibility issues: It Takes Two
- Steam AppID of the game: 1426210

## System Information
- GPU: AMD Radeon RX 570 4GB
- Video driver version: amdgpu
- Kernel version: 6.8.0-76060800daily20240311-generic
- Link to full system information report as Gist: [system report](https://gist.github.com/legoraft/8f878d305a44dddc1bcb5f8df89ea0e4)
- Proton version: experimental-9.0-20240419b

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

Game launch log: 
[steam-1426210.log](https://github.com/ValveSoftware/Proton/files/15141921/steam-1426210.log)

## Symptoms 
Launching the game goes perfectly, but when trying to play in online mode with another computer, my account doesn't show   up as online for the other player (who uses windows). My own in game overlay also doesn't show any friends, even though both players are friends within the ea app (checked this by running ea app through proton).

My EA account is linked to my steam account and this worked flawlessly a few months ago. I've tried removing and re-adding friends, running proton 8.0-5 and 7.0-6 and reinstalling the full game. None of which worked.

## Reproduction
1. Launch It Takes Two from the steam library on both computers
2. Once launched, go to online play and press 'invite friend'
3. No friends show up, even though they're added and show up in the normal ea app
4. Try to send a friend request (from linux computer) with the ingame overlay (press Shift + F1)
5. Friend request sometimes arrives on the other (windows) computer, but nothing happens when accepted.
6. Sending friend requests the other way won't work because player is already added.

**Comment by Leopard1907** [comment]
Game doesn't boot, dies after "Preparing Game" said by Ea Launcher.

- Proton Experimental and 9, both fails the same way
- 7900 XTX with RADV as gpu
- 7800x3D cpu
- Arch Linux- Kernel 6.8.9
- Resizable bar enabled

```
AMD_DEBUG=info glxinfo | grep vram
    vram_size = 24576 MB
    vram_vis_size = 24560 MB
    vram_type = 9
    has_dedicated_vram = 1
    all_vram_visible = 1
```

[steam-1426210.log](https://github.com/ValveSoftware/Proton/files/15341167/steam-1426210.log)

**Comment by Leopard1907** [comment]
Game got an update today that made it Deck Verified and removed EA Launcher from game.

https://store.steampowered.com/news/app/1426210/view/4150707902513320084

Game works now.

**Comment by MarcoSin42** [comment]
Initial setup causes game to crash.  In particular, when the game prompts the user for "Usage data" permission, clicking either "enable" or "disable" results in a fatal error.  
[steam-1426210.log](https://github.com/user-attachments/files/17142330/steam-1426210.log)

**Comment by parkerlreed** [comment]
Just installed and ran with Proton Experimental and was able to get past all the intro screens fine.

**Comment by MarcoSin42** [comment]
Just ran it with Proton Experimental.  Same issue.
[steam-1426210.log](https://github.com/user-attachments/files/17154303/steam-1426210.log)

**Comment by kisak-valve** [comment]
It Takes Two: Generic UE4 "Fatal Error!"

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8403.
@jross365 posted on 2025-01-20T21:55:47:

# Compatibility Report
- Name of the game with compatibility issues: **It Takes Two**
- Steam AppID of the game: **1426210**

## System Information
- GPU: **Nvidia RTX 3080**
- Video driver version: **550.120**
- Kernel version: **6.8.0-51-generic x86_64**
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.githubusercontent.com/jross365/67fab00d70695ed1f79aa3d90a3ca1c8/raw/b580fdad2ef1db53085d1e800f7cd1f6c9948176/gistfile1.txt
- Proton version: 9.0-4, 7.0-6, Proton Experimental

## I confirm:
- [Y] that I haven't found an existing compatibility report for this game.
- [Y] that I have checked whether there are updates for my system available.

Proton AppID log (**steam-1426210.log**) :  https://gist.githubusercontent.com/jross365/fd300f2d66fb3a25d4833b3d1ddd69f0/raw/12c4af950b2e9dc892644f6b1f14ce2ce5eb8a3e/gistfile1.txt

## Symptoms 
Game launches and plays the intro video, with sound. When prompted in the game with "Press any button", the Sound and Privacy Settings prompt is presented. After clicking either option in the Privacy Settings prompt, the game crashes with a UE4 "[Fatal Error!](https://github.com/jross365/MyProtonIssues/blob/main/ItTakesTwo/UE4-FatalError.png)"

To try to circumvent this prompt, I copied over a previous save/settings folder from my Windows AppData\Local\\ItTakesTwo folder. This bypassed the setup, but the game still crashed the same way when I pressed a button continue the game.

In the WINE environment, UE4 generates detailed crash reports such as (this latest one](https://github.com/jross365/MyProtonIssues/tree/main/ItTakesTwo/UE4CC-Windows-0EE5BF24460D83ECF1A588ACF2D358E6_0000) . The crash contains a  process memory dump and detailed XML of the UE4 crash condition.

## Reproduction
I have this result with every possible combination of Proton listed above, and with other flags (as recommended on the [ProtonDB page](https://www.protondb.com/app/1426210).

I have tried retrieving the previous game version (May 25, 2024) from manifest information gathered from SteamDB.info, which resulted in the exact same crash.

**Comment by jross365** [comment]
It Takes Two: Generic UE4 "Fatal Error!"

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8403.
@jross365 posted on 2025-01-20T21:55:47:

# Compatibility Report
- Name of the game with compatibility issues: **It Takes Two**
- Steam AppID of the game: **1426210**

## System Information
- GPU: **Nvidia RTX 3080**
- Video driver version: **550.120**
- Kernel version: **6.8.0-51-generic x86_64**
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.githubusercontent.com/jross365/67fab00d70695ed1f79aa3d90a3ca1c8/raw/b580fdad2ef1db53085d1e800f7cd1f6c9948176/gistfile1.txt
- Proton version: 9.0-4, 7.0-6, Proton Experimental

## I confirm:
- [Y] that I haven't found an existing compatibility report for this game.
- [Y] that I have checked whether there are updates for my system available.

Proton AppID log (**steam-1426210.log**) :  https://gist.githubusercontent.com/jross365/fd300f2d66fb3a25d4833b3d1ddd69f0/raw/12c4af950b2e9dc892644f6b1f14ce2ce5eb8a3e/gistfile1.txt

## Symptoms 
Game launches and plays the intro video, with sound. When prompted in the game with "Press any button", the Sound and Privacy Settings prompt is presented. After clicking either option in the Privacy Settings prompt, the game crashes with a UE4 "[Fatal Error!](https://github.com/jross365/MyProtonIssues/blob/main/ItTakesTwo/UE4-FatalError.png)"

To try to circumvent this prompt, I copied over a previous save/settings folder from my Windows AppData\Local\\ItTakesTwo folder. This bypassed the setup, but the game still crashed the same way when I pressed a button continue the game.

In the WINE environment, UE4 generates detailed crash reports such as [this latest one](https://github.com/jross365/MyProtonIssues/tree/main/ItTakesTwo/UE4CC-Windows-0EE5BF24460D83ECF1A588ACF2D358E6_0000). The crash contains a  process memory dump and detailed XML of the UE4 crash condition.

## Reproduction
I have this result with every possible combination of Proton listed above, and with other flags (as recommended on the [ProtonDB page](https://www.protondb.com/app/1426210).

I have tried retrieving the previous game version (May 25, 2024) from manifest information gathered from SteamDB.info, which resulted in the exact same crash.

---

@MarcoSin42 I had the exact same issue as you. Did yours also generate a crash dump in the compatdata PFX path for C:\users\steam\AppData\Local\ItTakesTwo folder?

**Comment by kisak-valve** [comment]
Hello @jross365, instead of becoming a proxy editor for you, I've moved the transferred issue report into your comment so that you can make further adjustments at your discretion.

**Comment by modellking** [comment]
Hello there wizards of forcing games to follow thy commands on linux.
Please help me:
I have massive crackling audio issues within the levels in this game

Compatibility Report
Name of the game with compatibility issues: It Takes Two
Steam AppID of the game: 1426210

System Information
GPU: AMD Radeon RX 7700 XT (behind Thunderbolt PCI Tunnel) and AMD Radeon 780M (from AMD Ryzen 9 7940HS)
Kernel version: 6.14.5-300.fc42.x86_64 (Fedora 42)
Proton version: Proton Experimental

I confirm:

[x] that I have checked whether there are updates for my system available.

Symptoms
Game launches fine (with either graphics card), but I always get crackling audio issues when in levels and some movement occurs. This crackling also affect other applications sound.

Reproduction
Start game

Join Online or offline

Move around

Other
CPU is only saturated to 80% on 1-2 cores

**Comment by actondev** [comment]
# Compatibility Report
- Name of the game with compatibility issues: It Takes Two
- Steam AppID of the game: 1426210


## System Information
- GPU: Intel Mesa Intel(R) Iris(R) Xe Graphics (TGL GT2)
- Video driver version: 4.6 (Compatibility Profile) Mesa 25.1.1-arch1.2
- Kernel version: 6.12.30-1-lts
- Link to full system information report as [Gist](https://gist.github.com/actondev/da40676cb660b341e61623e923c052c5):
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 

While the game working in the past (last played on march 29), trying to play it now I'm seeing weird shader/textures artifacts. This is visible also in the start screen:


- normal (vulkan): it seems like there's water (!?) in the scene, but in general it looks like a broken collage. Sometimes briefly (for a second or so) the correct screen is rendered in the beginning but then it looks broken again
![Image](https://github.com/user-attachments/assets/f82a9dd3-951f-4326-a498-e8f26b260958)

- `PROTON_USE_WINED3D11`: more similar to the correct start screen, but still wrong
![Image](https://github.com/user-attachments/assets/afbfc23e-deaf-4973-8d3e-ffd5640df8e6)


The attached log is with the normal invocation
[steam-1426210.log](https://github.com/user-attachments/files/20540446/steam-1426210.log)


## Update 2025-06-15
It seems to work fine once more. Still not sure if this was fixed cause of a system update or steam update. These are the changes in my system information https://gist.github.com/actondev/da40676cb660b341e61623e923c052c5/revisions

**Comment by andrewathalye** [comment]
I’m also experiencing this same graphical issue on Proton Experimental. It seems like there might be something with the 2025-06 Mesa release?

**Comment by yigitsalar** [comment]
# Compatibility Report
- Name of the game with compatibility issues: It Takes Two
- Steam AppID of the game: 1426210

## System Information
- GPU:  RX 580 TITAN OC Edition
- Video driver version:  4.6 (Compatibility Profile) Mesa 25.2.6-arch1.1
- Kernel version:  Zen 6.17.7.zen1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/yigitsalar/1f2dbb84086a7924953dae05ade1cbcd
- Proton version: Multiple Proton versions (Latest Proton-GE, Experimental, 9.0-4, UMU-Proton-9.0-4e, GE-Proton10-15)
## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.



## Symptoms 
Game crashes with "Fatal error!" after setting up the settings, or after pressing a key if the setup was completed successfully prior.

## Reproduction
1. Open the game
2. Press any key
3. Go through 2-page setup process
4. After you press "Decline" or "Accept" on data collection screen, the game will crash with "Fatal error!"

[steam-1426210.log](https://github.com/user-attachments/files/23496889/steam-1426210.log)

**Comment by runar-work** [comment]
I didn't run into any issues when I tested with RX 470 with Mesa 25.2.6, 25.2.7 and main `108d2d29`, with Proton Experimental on 6.17.8-arch1-1, so it doesn't seem to be a general Polaris issue, at least.

**Comment by yigitsalar** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4696#issuecomment-3521108136

#### Reports on ProtonDB
My compatibility report: https://www.protondb.com/app/1426210#7GE479UwBp
Other compatibility reports: https://www.protondb.com/app/1426210#Zy9i6Jygkl, https://www.protondb.com/app/1426210#YWaz-D8rXY

**Comment by yigitsalar** [comment]
I have solved it for myself.

### TL;DR:
I removed `ipv6.disable=1` parameter from the kernel and the error message no longer appears.

### Long answer

I have `ipv6.disable=1` parameter available on my kernel, which disables IPv6 stack, because [here on Türkiye, IPv6 protocol is uncommon, ISPs doesn't provide it or rarely do](https://www.ipv6-test.com/stats/country/TR). IPv6 delays every downloads with cURL, Wget, Git etc. by 10 seconds and it pisses me off, therefore I use it to disable IPv6.
Yesterday, I needed to use Parsec, which requires IPv6 to be enabled even if I don't use it. Then I enabled it for one boot to use Parsec temporarily. Before rebooting, I got reminded by this issue, then I tried and it work. I immediately grabbed logs for future in case it stops working. Today, in the morning, I tested it and saw that it didn't work. An hour ago, my friend talked about It Takes Two while Parsec is on my mind, then I immediately thought "What if I just enable IPv6 again?". Well, it worked.
I don't know if it will stay the same afterwards, but I can AT LEAST point the root cause in some form, I couldn't even do that before!

(EDIT: We beat the game without any "fatal error!", and I solved that IPv6 issue by disabling IPv6 protocol[s] on the router.)

**Comment by jross365** [comment]
@yigitsalar I have re-installed `It Takes Two` and confirmed that removing IPv6 disablement from my GRUB boot options resolved the crash issue for me, as well.

I edited my GRUB config (`sudo nano /etc/default/grub`).

In the GRUB config's `GRUB_CMDLINE_LINUX_DEFAULT` line, I commented out `ipv6.disable=1` :

<img width="1016" height="265" alt="Image" src="https://github.com/user-attachments/assets/6ed24485-2dda-4258-8beb-5f49e7693bf6" />

I saved the change, quit out of `nano`,  ran `sudo update-grub` and then rebooted.

@yigitsalar Thank you for figuring this out! This is the only game I haven't been able to play since moving over from Windows to Linux Mint.  You made it possible 🙏 

@MarcoSin42 IIRC you have the same issue.

@kisak-valve Is there a process/procedure/ticket or other way to formally record that this specific issue has a potential user-side fix?

**Comment by yigitsalar** [comment]
For future reference to disable/remove IPv6 disabler for intermediates:
**[THIS BREAKTHROUGH IS NEW. BLOCKING OTHER NETWORK-RELATED STUFF MIGHT PREVENT THE GAME FROM RUNNING, TOO.]**
Before starting, I want to write that you are responsible for the things you do to anything. If you break your system, don't blame me.

This guide assumes that you are using GRUB installed to its default location. If you don't use GRUB, please refer to your bootloader's wiki/man page etc. about changing kernel parameters. If you installed GRUB outside of the default location, edit the folder structure accordingly for yourself.

### Remove for one boot
After you start the system, you should be greeted with GRUB screen. Instead of pressing enter to select kernel, press `e` to edit configuration. If you see `ipv6.disable=1` somewhere, locate it with the arrow keys and delete it. Make sure to not delete anything else. After done, press F10 to boot. Your IPv6 stack will work for only one boot. If you don't see `ipv6.disable=1`, this method can't help you run the game.

### To make it persistent
Open the file `/etc/default/grub` with root permissions. This is the file that your GRUB configuration is stored.
In `GRUB_CMDLINE_LINUX_DEFAULT` line, remove the `ipv6.disable=1` parameter. Make sure that you don't delete anything else. After that, save the file and close it.

To apply the changes on Debian-based distros, run `update-grub` as root. To apply the changes on Arch-based distros, run `grub-mkconfig -o /boot/grub/grub.cfg` as root. For other distros, please refer to its wiki page. On Debian-based distros, changes are applied automatically once a module (?) or bootloader updates.
